packages/github_cli.rb

```diff
@@ -33,10 +33,4 @@ def self.build
   def self.install
     system "install -Dm755 bin/gh #{CREW_DEST_PREFIX}/bin/gh"
   end
-
-  def self.postinstall
-    puts
-    puts "Type 'gh' to get started.".lightblue
-    puts
-  end
```

Comment:
What is wrong with this postinstall section?

Reply:
In any event, new version?  See https://github.com/cli/cli/archive/refs/tags/v1.8.1.tar.gz

Reply:
the postinstall section felt redundant and unnecessary. Github already has amazing documentation and if someone is installing github cli, they either are reading github's documentation or already know how to use the tool

Reply:
You should never assume the user is knowledgeable enough to know how to use the tool; especially if the package name is not the same as the executable name.